src/pages/lessons/[slug].tsx

```diff
@@ -581,6 +582,12 @@ const Lesson: React.FC<LessonProps> = ({
       <div className="container max-w-screen-lg py-8 md:py-12 lg:py-16">
         <div className="grid grid-cols-1 gap-8 divide-y lg:grid-cols-1 lg:gap-12 md:divide-transparent divide-gray-50">
           <div className="row-start-1 space-y-6 md:col-span-8 md:row-start-1 md:space-y-8 lg:space-y-10">
+            {lesson.state === 'RETIRED' && (
```

Comment:
Is it supposed to be all-caps here and lowercase up above?

<img width="314" alt="CleanShot 2022-12-15 at 10 23 59@2x" src="https://user-images.githubusercontent.com/694063/207914139-c54efd16-4b3e-425e-af16-c59378c0975d.png">

Reply:
lesson state was coming in all-caps for whatever reason

![Use Chrome Developer Tools to Emulate Different Mobile Devices egghead io 2022-12-15 at 4 20 53 PM](https://user-images.githubusercontent.com/6188161/207969656-20c4e8ce-d24e-4536-af0e-3844372a3b41.jpg)